go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22.0
 toolchain go1.23.2
 
```

Comment:
I don't expect this to be 100% right away, I'd like to get it merged so people can build  the missing chunks while I fill out missing attributes and do cleanups.